validation-test/stdlib/Index.swift.gyb

```diff
@@ -64,7 +64,11 @@ Index.test(
   for test in indexStepsFromTests.filter(
     {$0.limit == nil && $0.distance >= 0}
   ) {
-    let c = ${Collection}(Array(0..<10))
+    let max = 10
+    guard test.expectedOffset! < max else {
```

Comment:
A minor nitpick: there's already a `filter` call on line 65, it only feels natural to include this condition in there. What do you think, @austinzheng?